Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:alpine as build
+
+WORKDIR /build
+
+ADD ./ /build/
```

Comment:
Codacy found an issue: [Use COPY instead of ADD for files and folders](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6290476)